fabric-resource-loader-v1/src/main/java/net/fabricmc/fabric/api/resource/v1/reloader/ResourceReloaderKeys.java

```diff
@@ -19,43 +19,43 @@
 import net.minecraft.resources.Identifier;
 
 /**
- * This class contains default keys for various Minecraft resource reloaders.
+ * This class contains default keys for various Minecraft reload listener.
  *
  * @see net.minecraft.server.packs.resources.PreparableReloadListener
  */
 public final class ResourceReloaderKeys {
 	/**
-	 * Represents the application phase before Vanilla resource reloaders are invoked.
+	 * Represents the application phase before Vanilla reload listeners are invoked.
 	 *
-	 * <p>No resource reloaders are assigned to this identifier.
+	 * <p>No reload listeners are assigned to this identifier.
 	 *
-	 * @see net.fabricmc.fabric.api.resource.v1.ResourceLoader#addReloaderOrdering(Identifier, Identifier)
+	 * @see net.fabricmc.fabric.api.resource.v1.ResourceLoader#addListenerOrdering(Identifier, Identifier)
 	 */
 	public static final Identifier BEFORE_VANILLA = Identifier.fromNamespaceAndPath("fabric", "before_vanilla");
 	/**
-	 * Represents the application phase after Vanilla resource reloaders are invoked.
+	 * Represents the application phase after Vanilla reload listeners are invoked.
 	 *
-	 * <p>No resource reloaders are assigned to this identifier.
+	 * <p>No reload listeners are assigned to this identifier.
 	 *
-	 * @see net.fabricmc.fabric.api.resource.v1.ResourceLoader#addReloaderOrdering(Identifier, Identifier)
+	 * @see net.fabricmc.fabric.api.resource.v1.ResourceLoader#addListenerOrdering(Identifier, Identifier)
 	 */
 	public static final Identifier AFTER_VANILLA = Identifier.fromNamespaceAndPath("fabric", "after_vanilla");
 
 	private ResourceReloaderKeys() { }
 
 	/**
-	 * Keys for various client resource reloaders.
+	 * Keys for various client reload listeners.
 	 */
 	public static final class Client {
-		public static final Identifier BLOCK_ENTITY_RENDERERS = Identifier.withDefaultNamespace("block_entity_renderers");
-		public static final Identifier BLOCK_RENDER_MANAGER = Identifier.withDefaultNamespace("block_render_manager");
-		public static final Identifier CLOUD_CELLS = Identifier.withDefaultNamespace("cloud_cells");
-		public static final Identifier EQUIPMENT_MODELS = Identifier.withDefaultNamespace("equipment_models");
-		public static final Identifier ENTITY_RENDERERS = Identifier.withDefaultNamespace("entity_renderers");
-		public static final Identifier DRY_FOLIAGE_COLORMAP = Identifier.withDefaultNamespace("dry_foliage_colormap");
-		public static final Identifier FOLIAGE_COLORMAP = Identifier.withDefaultNamespace("foliage_colormap");
+		public static final Identifier BLOCK_ENTITY_RENDER_DISPATCHER = Identifier.withDefaultNamespace("block_entity_render_dispatcher");
```

Comment:
I have concerns with the value renames (not the field name itself).
Doing this may cause implicit breakage that a field rename doesn't as it would be an explicit breakage.

Anyone using a similar value but not the field directly will diverge, and this will not be caught by a compile error, which may be hard to find.

Reply:
This is a valid concern that we should discuss, but I would like to merge this PR to make whole process easier. I think the best way forward will be to review the main PR.